src/content/blog/2025-08-01-four-datasets-that-impact-your-life-every-day.mdx

```diff
@@ -0,0 +1,80 @@
+
+In fact, all of the datasets we've talked about so far--address points, address system quadrants, and roads--are all maintained as a joint effort between UGRC, local governments, and fellow state agencies. This collaboration is at the heart of everything we do at UGRC.
+
+So far, all the datasets we've covered have also been vector datasets--which means they are discrete shapes in the form of points (address points), lines (roads) and polygons (address system quadrants). But what about raster data? How does that affect our day-to-day lives?
+
+## Discover Basemap and Imagery Server
+
+UGRC's [Discover server](/products/discover/) just celebrated ten years of activity earlier this year! Discover is our one-stop shop for aerial imagery and custom Utah basemap services, all available from UGRC for free to public and licensed users. Our nearly 2,000 registered accounts include all kinds of government agencies from the Utah Department of Transportation to the Utah Division of Wildlife Resources. Even if you've never used Discover, the imagery and basemaps available on the server are used in all kinds of planning operations that impact many areas of Utah life, such as:
+
+- Drinking water budgeting and distribution
+- Transportation safety and planning
+- Rural public land issues
+- Broadband access expansion
+- 911 and emergency response
+- Natural resources management
+- Infrastructure/Asset Management
+
+These are just a few of the possible use cases for statewide imagery. You can learn more about what private organizations, government agencies, and others are doing with the power of Discover on [our blog](/blog/2025-05-05-celebrating-ten-years-of-discover/).
+
+## There's Always More Data
+
+While we've picked out just a few datasets to chat about, there's many more that impact daily life. There's the [Public Safety Answering Point](/products/sgid/society/public-safety-answering-point-boundaries/) (PSAP) boundaries, which are used to make sure your 911 call goes to the correct dispatch center. There's all our [parcel](/products/sgid/cadastre/parcels/) and [land ownership data](/products/sgid/cadastre/land-ownership/), which are used for all kinds of analytics and operations planning. And beyond these, there's so many other 'little brown data birds' that elevate the quality of life for residents in Utah. These datasets, along with more than 300 other unique layers, can be found in the [Statewide Geographic Information Datasource](/products/sgid/), or SGID. You can browse these datasets using our [SGID Index](/products/sgid/sgid-index/) or by viewing our array of [data categories](/products/sgid/categories/). For questions or comments regarding SGID data, don't hesitate to reach out to [our team](/contact/).
```

Comment:
I would suggest restraining from calling any of this data "ours". 

Our parcel data and our land ownership data implies that we created it or somehow own it when we simply aggregate or host it. I wouldn't want to give the wrong impression or hurt anyones feelings by claiming/insinuating ownership.